doc/tutorials/using_partition_mode_on_up2.rst

```diff
@@ -167,133 +165,74 @@ Enable partition mode in ACRN hypervisor
-           .bar = {
-              [0] = {
-                .base = 0UL,
-                .size = 0x2000UL,
-                .type = PCIBAR_MEM32
-              },
-              [1] = {
-                .base = 0UL,
-                .size = 0x1000UL,
-                .type = PCIBAR_MEM32
-              },
-              [5] = {
-                .base = 0UL,
-                .size = 0x1000UL,
-                .type = PCIBAR_MEM32
-              },
-           },
-           .pdev = {
-              .bdf.bits = {.b = 0x00U, .d = 0x12U, .f = 0x0U},
-              .bar = {
-              [0] = {
-                .base = 0x91514000UL,
-                .size = 0x2000UL,
-                .type = PCIBAR_MEM32
-              },
-              [1] = {
-                .base = 0x91537000UL,
-                .size = 0x100UL,
-                .type = PCIBAR_MEM32
-              },
-              [5] = {
-                .base = 0x91536000UL,
-                .size = 0x800UL,
-                .type = PCIBAR_MEM32
-              },
-           }
-           ...
-           static struct vpci_vdev_array vpci_vdev_array2 = {
-              .num_pci_vdev = 3,
-              ...
-              {/*vdev 1: USB controller*/
-                 .vbdf.bits = {.b = 0x00U, .d = 0x01U, .f = 0x0U},
-                 .ops = &pci_ops_vdev_pt,
-                 .bar = {
-                    [0] = {
-                      .base = 0UL,
-                      .size = 0x10000UL,
-                      .type = PCIBAR_MEM32
-                    },
-                 },
-                 .pdev = {
-                    .bdf.bits = {.b = 0x00U, .d = 0x15U, .f = 0x0U},
-                    .bar = {
-                      [0] = {
-                        .base = 0x91500000UL,
-                        .size = 0x10000UL,
-                        .type = PCIBAR_MEM64
-                      },
-                    }
-              ...
-
-#. Optionally, configure the ``.bootargs`` kernel command line arguments
+     struct acrn_vm_pci_ptdev_config vm0_pci_ptdevs[2] = {
+	{
+		.vbdf.bits = {.b = 0x00U, .d = 0x00U, .f = 0x00U},
+		.pbdf.bits = {.b = 0x00U, .d = 0x00U, .f = 0x00U},
+	},
+	{
+		.vbdf.bits = {.b = 0x00U, .d = 0x01U, .f = 0x00U},
+		.pbdf.bits = {.b = 0x00U, .d = 0x12U, .f = 0x00U},
+	},
+     };
 
+     ...
+     struct acrn_vm_pci_ptdev_config vm1_pci_ptdevs[3] = {
+	{
+		.vbdf.bits = {.b = 0x00U, .d = 0x00U, .f = 0x00U},
+		.pbdf.bits = {.b = 0x00U, .d = 0x00U, .f = 0x00U},
+	},
+	{
+		.vbdf.bits = {.b = 0x00U, .d = 0x01U, .f = 0x00U},
+		.pbdf.bits = {.b = 0x00U, .d = 0x15U, .f = 0x00U},
+	},
+	{
+		.vbdf.bits = {.b = 0x00U, .d = 0x02U, .f = 0x00U},
+		.pbdf.bits = {.b = 0x02U, .d = 0x00U, .f = 0x00U},
+	},
+     };
+     ...
+
+#. Optionally, configure the ``VMx_CONFIG_OS_BOOTARGS`` kernel command line arguments
+
+   The partiton mode configure infomation is located in header
+   file ``hypervisor/arch/x86/configs/apl-mrb/partition_config.h``.
```

Comment:
Why is this file under `apl-mrb`? Could we promote it to `hypervisor/arch/x86/configs` if it's applicable to other platforms?